torch/_dynamo/side_effects.py

```diff
@@ -993,7 +1013,7 @@ def codegen_update_mutated(self, cg: PyCodegen):
                     else:
                         cg.tx.output.update_co_names(name)
                         cg(value)
-                        cg(var.source)
+                        cg(var)
```

Comment:
Now we are tracking side effects for newly constructed `TensorVariable` as well, but they don't have a source (see added comments above). However, the `cg` calls here don't need to explicitly call on `source` anyways, since we are just trying to codegen the object representation of this `var`; if it has a source, it'll automatically use it, and if it doesn't, like the `TensorVariable` case, it'll go down whatever special handling we have in `PyCodegen.__call__`.